pkg/provider/helm/values.go

```diff
@@ -86,10 +84,16 @@ func (g *HelmValuesGenerator) GenerateValues() (map[string]any, error) {
 		return nil, err
 	}
 
+	sdsConfig, err := getSDSConfig(tz.TrustZoneProto.GetProfile())
```

Comment:
Just for consistency with surrounding code:
```suggestion
	sdsConfig, err := getSDSConfig(g.trustZone.GetProfile())
```